.bumpversion.cfg

```diff
@@ -1,8 +0,0 @@
-[bumpversion]
-current_version = 0.3.1
-commit = True
-tag = True
-tag_name = {new_version}
-
-[bumpversion:file:nbclient/_version.py]
-
```

Comment:
Was it intentional to remove this file?

Reply:
nope :( sorry about that, must have done the rebase wrong...

thanks for catching this.